File: app/src/main/java/dev/ragnarok/fenrir/api/model/response/VkReponse.java
package dev.ragnarok.fenrir.api.model.response;

import androidx.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import dev.ragnarok.fenrir.api.model.Error;


public class VkReponse {

    @Nullable
    @SerializedName("error")
    public Error error;

    @Nullable
    @SerializedName("execute_errors")
    public List<Error> executeErrors;
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/response/VkReponse.kt
package dev.ragnarok.fenrir.api.model.response

import com.google.gson.annotations.SerializedName
import dev.ragnarok.fenrir.api.model.Error

open class VkReponse {
    @SerializedName("error")
    var error: Error? = null

    @SerializedName("execute_errors")
    var executeErrors: List<Error>? = null
}